Application/FoxFlss/data/Makefile.am

```diff
@@ -1,87 +1,33 @@
-# Directory for user-enabled tools
+ACLOCAL_AMFLAGS = -I m4
+
 FoxFlss_dir = /opt/foxconn
-FoxFlss_datadir = $(FoxFlss_dir)/data
-FoxFlss_userdir = $(FoxFlss_dir)/fcc-unlock.d
-FoxFlss_suspdir = $(FoxFlss_dir)/suspend
+FoxFlss_rfdatadir = $(FoxFlss_dir)/data
+FoxFlss_fccunlockdir = $(FoxFlss_dir)/fcc-unlock.d
+FoxFlss_suspenddir = $(FoxFlss_dir)/suspend
+FoxFlss_modemmanagerservicedir = $(sysconfdir)/systemd/system/ModemManager.service.d
 
-FoxFlss_data_SCRIPTS = \
+FoxFlss_rfdata_DATA = \
```

Comment:
rfdata is expected to have sufficient permissions to write the data inside and is set to 755.
And we need to set $(prefix), when the prefix parameter to a **local path**, "make install" and
"make uninstall" do not require root privileges.

Reply:
1. 755 means executable, 644 means non-executable; both 755 and 644 are writable by file owner.  As the host system is not be capable to run the RF binary in the OS, I believe 644 is fine.

2. If you want to install into a local directory (for example, installing into a chroot, or install into a packaging directory and pack into deb file), you can use DESTDIR parameter; this points to the root directory of the target environment, and does not require root to run.  That is what our builder does:

```
cd Application && make -j8 install DESTDIR=/<<PKGBUILDDIR>>/debian/tmp AM_UPDATE_INFO_DIR=no
```